### ISSUE TITLE
enhancement: fix TOC to bottom on mobile

### ISSUE THREAD

**Issue by chucklorenz** [cgkineo/adapt-toc#7]
Just some thoughts for future development in light of our use of thumbs.

- pin TOC to the bottom
- animate TOC up from the bottom
- move close btn to bottom or eliminate completely due to close on touch
- move top navbar to the bottom entirely to improve convenience
![image](https://user-images.githubusercontent.com/9489014/164910874-fd1bf7e4-5a54-4aa4-ab5f-894518bec268.png)

**Comment by danielghost** [comment]
Check if drawer attributes for alignment handle this.

**Comment by danielghost** [comment]
As default, the drawer fills the height regardless of whether any overflow is needed. The drawer position attributes don't currently account for vertical alignment.

Is this something we think is worth pursuing, or are we happy for this to remain as is? If this was to change, I assume other plugins using drawer would require the same options and therefore it would be something we change in core instead.

**Comment by danielghost** [comment]
Agreed not to action bottom positioning for items in the drawer. Not all mobile users will use single handed interactions using thumbs and it was mentioned that bottom positioning can cause rendering issues with the overflow.

The ability to add navigation to the bottom was included in https://github.com/adaptlearning/adapt-contrib-core/pull/166.
